test/attr/Inputs/access-note-gen.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+# coding=utf8
+
+"""
+Convert selected @objc attributes in a source file into access notes, removing
+the originals in the process.
+"""
+
+from __future__ import print_function
+
+import io
+import re
+import sys
+
+#
+# Entry point
+#
+
+
+def main():
+    if len(sys.argv) != 4:
+        print('Too few args to ' + sys.argv[0])
+        print('Usage: access-note-gen.py <input-file> <output-source-file> ' +
+              '<output-access-notes-file>')
+        sys.exit(1)
+
+    with io.open(sys.argv[1], mode='r', encoding='utf8') as input_file, \
+         io.open(sys.argv[2], mode='w', encoding='utf8') as output_file, \
+         io.open(sys.argv[3], mode='w', encoding='utf8') as access_notes_file:
+
+        # Add header to access notes file
+        access_notes_file.write(u"""\
+Reason: 'fancy tests'
+Notes:""")
+
+        # Loop over input lines, transforming them into output lines, writing access
+        # notes as a side effect.
+        for input_line in input_file:
+            # Look for access-note-move comments.
+            input_line = access_note_move_re.sub(replacer(move_at_objc_to_access_note,
+                                                          access_notes_file),
+                                                 input_line, count=1)
+
+            # Look for access-note-adjust comments.
+            input_line = adjust_comment_re.sub(replacer(adjust_comments),
+                                               input_line, count=1)
+
+            output_file.write(input_line)
+
+
+#
+# Offsets
+#
+
+"""Matches an @±N offset."""
+offset_re_fragment = r'[ \t]*(?:@([+-]\d+))?[ \t]*'
+
+
+def offsetify(*offsets):
+    """Sum line offsets matched by offset_fragment_re and convert them to strings
+       like @+3 or @-2."""
```

Comment:
Should this reference `offsets` or `offset_re_fragment` instead of `offset_fragment_re`?

Reply:
You're right, it should say `offset_re_fragment`.